deploy.js

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env node
+
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
This is actually used for all files. ESLint expect that all `require()`s are production dependencies. This comment makes an exception to this rule. It basically says: I'm going to `require()` a dev dependency because I know that this file is only used in a development environment.